README.md

```diff
@@ -39,6 +39,31 @@ rm docs/rook/master
 git checkout -- docs/rook/master
 ```
 
+## Test using a container
+
+### Run a jekyll container
+
+* On a SELinux enabled OS:
+
+    ```console
+    cd rook.github.io
+    sudo docker run -d --name rookio -p 4000:4000 -v $(pwd):/srv/jekyll:Z jekyll/jekyll jekyll serve --watch
```

Comment:
Why do we need sudo? For the jekyll mount?

Reply:
If I recall correctly is for the 'Z' in the jekyll mount that does the relabelling on the provided folder so that it can be accessed by the container

Reply:
@iranzo You should only need `sudo` if your user is not in the, e.g., `docker` group, depending on the OS. The `docker` group allows non-root users access to the Docker Daemon socket (`/var/run/docker.sock` or so).

I would vote to remove the `sudo`.

Reply:
@iranzo You should only need `sudo` if your user is not in the, e.g., `docker` group, depending on the OS. The `docker` group allows non-root users access to the Docker Daemon socket (`/var/run/docker.sock` or so).

I would vote to remove the `sudo`.

Reply:
Removed on both cases